SingularityUI/app/collections/RequestHistoricalTasks.coffee

```diff
@@ -1,25 +0,0 @@
-RequestTasks = require './RequestTasks'
-PaginableCollection = require './PaginableCollection'
-TaskHistoryItem = require '../models/TaskHistoryItem'
-
-class HistoricalTasks extends PaginableCollection
-
-    model: TaskHistoryItem
-
-    url: -> "#{ config.apiRoot }/history/request/#{ @requestId }/tasks"
-
-    initialize: (models, {@requestId}) ->
-
-    comparator: (task0, task1) =>
-        -(task0.get("updatedAt") - task1.get("updatedAt"))
-
-    parse: (data) ->
-        for task in data
-            task.id = task.taskId.id
-        data
-
-    getTasksForDeploy: (deployId) ->
```

Comment:
didn't see it anywhere, but assuming this method is unused since you didn't reimplement it, is that correct?